build.gradle

```diff
@@ -101,7 +101,7 @@ allprojects {
   repositories {
     jcenter()
     mavenCentral()
-    //TODO: Update the cloudsmith maven repo once created - for internal libraries such as Signers
+    maven { url "https://artifacts.consensys.net/public/maven/maven/" }
     maven { url "https://consensys.bintray.com/pegasys-repo" }
```

Comment:
Is the bintray repo still needed?

Reply:
hmm, I can attempt to remove it and run the build .. good idea.